orchestrators/ProcurementOperations/src/Coordinators/VendorRecommendationCoordinator.php

```
@@ -24,69 +34,352 @@ public function __construct(
+                    return $this->unavailableResult($request, 'provider_output_rejected');
+                }
+
+                $fitScore = max(0, min(100, $fitScore + $delta));
             }
 
-            $candidates[] = $this->applyEnrichment($candidate, $candidateEnrichment);
+            $eligibleCandidates[] = new VendorRecommendationEligibleCandidate(
+                vendorId: $deterministicCandidate->vendorId,
+                vendorName: $deterministicCandidate->vendorName,
+                fitScore: $fitScore,
+                confidenceBand: VendorRecommendationScoredCandidate::confidenceBandFor($fitScore),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Nexus\ProcurementOperations\DTOs\VendorRecommendation\VendorRecommendationScoredCandidate' in method 'recommend'. (undefined)

(StaticAccess)